sandbox/public/index.html

```diff
@@ -58,6 +58,9 @@
         configId: "9999999",
         orgId: "53A16ACB5CC1D3760A495C99@AdobeOrg",
         debugEnabled: true,
+        idMigrationEnabled: !(
+          location.href.indexOf("idMigrationEnabled=false") >= 0
+        ),
```

Comment:
Did you intend to leave this here? If so, you can use the `getUrlParameter` function to get the value of the `idMigrationEnabled` query param.